gazebo_ros/test/plugins/args_init.cpp

```diff
@@ -56,7 +56,8 @@ void ProperInit::Load(int argc, char ** argv)
   auto node = gazebo_ros::Node::Get();
 
   // Create a publisher
-  auto pub = node->create_publisher<std_msgs::msg::String>("test", rclcpp::SystemDefaultsQoS());
+  auto pub = node->create_publisher<std_msgs::msg::String>("test",
+      rclcpp::QoS(rclcpp::KeepLast(1)).transient_local());
```

Comment:
I think uncrustify prefers breaking at the open parenthesis:
```suggestion
  auto pub = node->create_publisher<std_msgs::msg::String>(
    "test",
    rclcpp::QoS(rclcpp::KeepLast(1)).transient_local());
```

Same for other lines in this PR.

Reply:
thanks: f31628b